src/SDKs/Cdn/Cdn.Tests/ScenarioTests/CustomDomainTests.cs

```diff
@@ -112,17 +112,40 @@ public void CustomDomainCRUDTest()
                 // Start endpoint
                 cdnMgmtClient.Endpoints.Start(resourceGroupName, profileName, endpointName);
 
-                // Enable custom https on custom domain that is already enabled should fail
+                // Enable custom https using CDN managed certificate on custom domain 
                 cdnMgmtClient.CustomDomains.EnableCustomHttps(resourceGroupName, profileName, endpointName, customDomainName1);
 
-                // Delete first custom domain on stopped endpoint should succeed
+                // add the same deleted custom domain again
+                cdnMgmtClient.CustomDomains.Create(resourceGroupName, profileName, endpointName, customDomainName2, "sdk-2-f3757d2a3e10.DUSTYDOGPETCARE.US");
+
+                // Enable custom https using BYOC on custom domain 
+                var byocParameters = new KeyVaultCertificateSourceParameters()
+                {
+                    ResourceGroupName = "byoc",
+                    SecretName = "CdnSDKE2EBYOCTest",
```

Comment:
@nachakra no hardcoding secrets.
Please use TestFramework to inject custom data.
https://github.com/Azure/azure-sdk-for-net/blob/master/Documentation/Using-Azure-TestFramework.md#1-once-you-set-your-connection-string-you-can-add-or-update-keyvalue-settings

Reply:
This is the cert name in Keyvault and not a secret :). That's how the property is called in the API model.